subiquity/server/controllers/tests/test_install.py

```diff
@@ -221,23 +226,56 @@ async def test_install_package(self, m_sleep):
             with self.assertRaises(subprocess.CalledProcessError):
                 await self.controller.install_package(package="git")
 
-    def setup_rp_test(self):
+    def setup_rp_test(self, lsblk_output=b"lsblk_output"):
         app = self.controller.app
         app.opts.dry_run = False
         fsc = app.controllers.Filesystem
         fsc.reset_partition_only = True
         app.package_installer = Mock()
-        app.command_runner = Mock()
-        self.run = app.command_runner.run = AsyncMock()
+        app.command_runner = AsyncMock()
+        self.run = app.command_runner.run = AsyncMock(
+            return_value=subprocess.CompletedProcess((), 0, stdout=lsblk_output)
+        )
         app.package_installer.install_pkg = AsyncMock()
         app.package_installer.install_pkg.return_value = PackageInstallState.DONE
         fsm, self.part = make_model_and_partition()
 
     @patch("subiquity.server.controllers.install.get_efibootmgr")
-    async def test_create_rp_boot_entry_add(self, m_get_efibootmgr):
+    async def test_configure_rp_boot_uefi_add(self, m_get_efibootmgr):
+        m_get_efibootmgr.side_effect = iter([efi_state_no_rp, efi_state_with_rp])
+        self.setup_rp_test()
+        await self.controller.configure_rp_boot_uefi(rp=self.part)
+        calls = [
+            call(
+                [
+                    "efibootmgr",
+                    "--create",
+                    "--loader",
+                    "\\EFI\\boot\\shimx64.efi",
```

Comment:
According to https://cdimage.ubuntu.com/daily-live/current/mantic-desktop-amd64.list, there's no shimx64.efi in the recovery partition (which is a copy of the disk image content)

My test suggests that this boot entry won't appear in the boot selection.

Reply:
Oops, you're right of course, fixed hopefully. Tianocore's habit of booting anything that looks plausible strikes again :(